src/components/ConfirmDialogProvider.react.js

```diff
@@ -53,9 +52,9 @@ export default class ConfirmDialogProvider extends React.Component {
                     (loading_state && loading_state.is_loading) || undefined
                 }
             >
-                {realChild && realChild.length
-                    ? realChild.map(wrapClick)
-                    : wrapClick(realChild)}
+                {children && children.length
+                    ? children.map(c => wrapClick(c))
+                    : wrapClick(children)}
```

Comment:
The child is not nested -- operate on the tree container directly

Reply:
Why the change from `children.map(wrapClick)` to `children.map(c => wrapClick(c))`?

Also, not new but perhaps `Array.isArray(children)` would be better than `children && children.length`.

Reply:
Guessing I did a console.log at some point and didn't revert it back correctly. Reverting.